<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LevelEditorWindow</class>
 <widget class="QMainWindow" name="LevelEditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="sidebar" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTabWidget" name="sidebarTabWidget">
          <property name="currentIndex">
           <number>-1</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QScrollArea" name="levelViewArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>69</width>
          <height>556</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>957</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&lt;File&gt;</string>
    </property>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&lt;View&gt;</string>
    </property>
    <addaction name="actionToggleLayer1"/>
    <addaction name="actionToggleLayer2"/>
    <addaction name="separator"/>
    <addaction name="actionZoom_Maximum"/>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_100"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="actionZoom_Minimum"/>
    <addaction name="separator"/>
    <addaction name="actionFullscreen"/>
    <addaction name="actionGrid"/>
    <addaction name="actionLiquid"/>
    <addaction name="actionLemmy_Mode"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&lt;Edit&gt;</string>
    </property>
    <addaction name="actionPaste"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionRaise"/>
    <addaction name="actionLower"/>
    <addaction name="actionRaiseLayer"/>
    <addaction name="actionLowerLayer"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionPaste"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionRaise"/>
   <addaction name="actionLower"/>
   <addaction name="actionRaiseLayer"/>
   <addaction name="actionLowerLayer"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_In"/>
   <addaction name="actionZoom_100"/>
   <addaction name="actionZoom_Out"/>
   <addaction name="separator"/>
   <addaction name="actionFullscreen"/>
   <addaction name="actionGrid"/>
   <addaction name="actionLiquid"/>
   <addaction name="separator"/>
   <addaction name="actionToggleLayer1"/>
   <addaction name="actionToggleLayer2"/>
   <addaction name="separator"/>
   <addaction name="actionAddArea"/>
   <addaction name="actionDeleteCurrentArea"/>
  </widget>
  <action name="actionToggleLayer1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&lt;Layer 1&gt;</string>
   </property>
   <property name="toolTip">
    <string>&lt;Layer 1&gt;</string>
   </property>
  </action>
  <action name="actionToggleLayer2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&lt;Layer 2&gt;</string>
   </property>
   <property name="toolTip">
    <string>&lt;Layer 2&gt;</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="text">
    <string>&lt;Zoom In&gt;</string>
   </property>
   <property name="toolTip">
    <string>&lt;Zoom In&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="text">
    <string>&lt;Zoom Out&gt;</string>
   </property>
   <property name="toolTip">
    <string>&lt;Zoom Out&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionZoom_100">
   <property name="text">
    <string>&lt;Zoom 100%&gt;</string>
   </property>
   <property name="toolTip">
    <string>&lt;Zoom 100%&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionZoom_Maximum">
   <property name="text">
    <string>&lt;Maximum Zoom&gt;</string>
   </property>
   <property name="toolTip">
    <string>&lt;Maximum Zoom&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+PgDown</string>
   </property>
  </action>
  <action name="actionZoom_Minimum">
   <property name="text">
    <string>&lt;Minimum Zoom&gt;</string>
   </property>
   <property name="toolTip">
    <string>&lt;Minimum Zoom&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+PgUp</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&lt;Save&gt;</string>
   </property>
   <property name="toolTip">
    <string>&lt;Save&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>&lt;Paste&gt;</string>
   </property>
   <property name="toolTip">
    <string>&lt;Paste&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>&lt;Cut&gt;</string>
   </property>
   <property name="toolTip">
    <string>&lt;Cut&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>&lt;Copy&gt;</string>
   </property>
   <property name="toolTip">
    <string>&lt;Copy&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>&lt;Delete&gt;</string>
   </property>
   <property name="toolTip">
    <string>&lt;Delete&gt;</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&lt;Fullscreen&gt;</string>
   </property>
   <property name="toolTip">
    <string>&lt;Fullscreen&gt;</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&lt;Grid&gt;</string>
   </property>
   <property name="toolTip">
    <string>&lt;Grid&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionAddArea">
   <property name="text">
    <string>&lt;Add Area&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+N</string>
   </property>
  </action>
  <action name="actionDeleteCurrentArea">
   <property name="text">
    <string>&lt;Delete Current Area&gt;</string>
   </property>
   <property name="toolTip">
    <string>&lt;Delete Current Area&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+D</string>
   </property>
  </action>
  <action name="actionRaise">
   <property name="text">
    <string>&lt;Raise to Top&gt;</string>
   </property>
   <property name="toolTip">
    <string>&lt;Raise to Top&gt;</string>
   </property>
  </action>
  <action name="actionLower">
   <property name="text">
    <string>&lt;Lower to Bottom&gt;</string>
   </property>
   <property name="toolTip">
    <string>&lt;Lower to Bottom&gt;</string>
   </property>
  </action>
  <action name="actionRaiseLayer">
   <property name="text">
    <string>&lt;Raise Layer&gt;</string>
   </property>
   <property name="toolTip">
    <string>&lt;Raise Layer&gt;</string>
   </property>
  </action>
  <action name="actionLowerLayer">
   <property name="text">
    <string>&lt;Lower Layer&gt;</string>
   </property>
   <property name="toolTip">
    <string>&lt;Lower Layer&gt;</string>
   </property>
  </action>
  <action name="actionLiquid">
   <property name="text">
    <string>Toggle Liquid</string>
   </property>
   <property name="toolTip">
    <string>Toggle Liquid</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionLemmy_Mode">
   <property name="text">
    <string>Lemmy Mode</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
